File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        HashSet<Integer> set = new HashSet<>();
        for(int v : nums){
            set.add(v);
        }
        ArrayList<Integer> sett = new ArrayList<>(set);
        Collections.sort(sett);
        int pointer=0;
        for(int v : sett){
            nums[pointer++] = v;
        }
        return pointer;
        
    }
}

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution(object):
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        list = set()
        for num in nums:
            list.add(num)
        list = sorted(list)
        for i, num in enumerate(list):
            nums[i] = num
        return len(list)
